chapters/background.tex

```diff
@@ -1,4 +1,11 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+TensorFlow \cite{tensorflow} is a open source machine learning library developed by Google.
+It supports a wide variety of programming languages, mainly targeting web and mobile applications.
+For mobile development TensorFlow provides a Lite version called TensorFlow Lite (TFLite) which can be easily included in a mobile app.
+The main advantage TFLite offers, is the ability to add a self trained model to detect smiles.
```

Comment:
I would like to have some bridge like this here:
Regarding the smile detection of soHappy, Tensor flow appears to be a convenient candidate.